content/en/docs/contribute/style/style-guide.md

```diff
@@ -111,10 +111,11 @@ document, use the backtick (`).
   <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
   <tr><td>For declarative management, use <code>kubectl apply</code>.</td><td>For declarative management, use "kubectl apply".</td></tr>
   <tr><td>Enclose code samples with triple backticks. <code>(```)</code></td><td>Enclose code samples with any other syntax.</td></tr>
+  <tr><td>Use single backticks to enclose inline code. For example, `var example = true`.</td><td>Use two asterisks (**) or an underscore (_) backticks to enclose inline code. For example, **var example = true**.</td></tr><tr><td>Use triple backticks before and after a multi-line block of code for fenced code blocks.</td><td>Use multi-line blocks of code to create diagrams, flowcharts, or other illustrations.</td></tr><tr><td>Use meaningful variable names that have a context.</td><td>Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.</td></tr><tr><td>Remove trailing spaces in the code.</td><td>Add trailing spaces in the code, because the screen reader will readout the spaces as well.</td></tr>
 </table>
 
 {{< note >}} 
-The website supports syntax highlighting for code samples, but specifying a language is optional.
+The website supports syntax highlighting for code samples, but specifying a language is optional. Syntax highlighting in the code block should conform to the [contrast guidelines.](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=141%2C143#contrast-minimum)
```

Comment:
- Is the language attribute optional or should authors make their best effort to set this attribute? I would rather have the language attribute set for code blocks.
- Do the k8s docs conform to this guideline? I am not sure. How can we check? Personally, I think the syntax highlighting could be improved, especially for the YAML code blocks.

Reply:
> * Is the language attribute optional or should authors make their best effort to set this attribute? I would rather have the language attribute set for code blocks.
> * Do the k8s docs conform to this guideline? I am not sure. How can we check? Personally, I think the syntax highlighting could be improved, especially for the YAML code blocks.

This has always been optional, please see https://kubernetes.io/docs/contribute/style/style-guide/#inline-code-formatting. I only added information about the guidelines for color contrast. The link that I provided has all the information and methods that can be used to check. This is a guideline and not a requirement.